tests/test_spaces.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+from stable_baselines3 import A2C, PPO
+from stable_baselines3.common.identity_env import IdentityEnvMultiBinary, IdentityEnvMultiDiscrete
+from stable_baselines3.common.evaluation import evaluate_policy
+from stable_baselines3.common.vec_env import DummyVecEnv
+
+
+MODEL_LIST = [A2C, PPO]
+DIM = 3
+
+
+@pytest.mark.parametrize("model_class", MODEL_LIST)
+def test_identity_multidiscrete(model_class):
+    """
+    Test if the algorithm (with a given policy)
+    can learn an identity transformation (i.e. return observation as an action)
+    with a multidiscrete action space
+    :param model_class: (BaseRLModel) A RL Model
+    """
+    env = DummyVecEnv([lambda: IdentityEnvMultiDiscrete(DIM)])
+
+    model = model_class("MlpPolicy", env, gamma=0.5, seed=0)
+    model.learn(total_timesteps=1000)
+    evaluate_policy(model, env, n_eval_episodes=5)
+    obs = env.reset()
+
+    evaluate_policy(model, env, n_eval_episodes=20, reward_threshold=80)
```

Comment:
I'm curious: what is the performance achieved?
btw, you can also merge the two tests, no?

Reply:
With DIM=3, for PPO:
multicategorical (89.25, 3.9607449)
multibinary (100.0, 0.0)

For A2C:
multicategorical (100.0, 0.0)
multibinary (100.0, 0.0)

Reply:
then give it a little bit more training steps set the threshold to 90 ;)
those are easy to solve env, and the test should almost fail (so that if we introduce a bug, it won't work as good as before)